quantities/markup.py

```diff
@@ -100,7 +100,7 @@ def format_units_latex(udict,font='mathrm',mult=r'\\cdot',paren=False):
     By default this is the latex \\cdot symbol.  Other useful
     options may be '' or '*'.
 
-    If paren=True, encapsulate the string in '\\left(' and '\right)'
+    If paren=True, encapsulate the string in '\\left(' and '\\right)'
```

Comment:
Only one of the two errors on this line had been fixed

https://github.com/python-quantities/python-quantities/blob/269d81e8f1fa308b5a0701b8136fd642153e9534/quantities/markup.py#L103